recipes/thuner/meta.yaml

```
@@ -0,0 +1,82 @@
+{% set name = "thuner" %}
+{% set version = "0.0.10" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f8ae60837da55fe699f519751f69a60af45e39cb8cb1a7b0b7f31a3b66be5036
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win]
```

The way to do version constraints with non noarch packages is:
```suggestion
  skip: True  # [win or py<310]
```

---

https://conda-forge.org/docs/maintainer/adding_pkgs/#skipping-builds

---

Hallelujah! It finally passed! Thanks for your help :)

---

No worries! Welcome to the `conda-forge` community :wave: